bans-core/src/main/java/space/arim/libertybans/core/selector/SelectionImpl.java

```diff
@@ -164,6 +165,18 @@ private Condition getPredication(SelectionOrder selection, PunishmentFields fiel
 				);
 			}
 		}
+		if (!seekBeforeStartTime.equals(Instant.EPOCH)) {
+			long seekBeforeId = selection.seekBeforeId();
+			if (seekBeforeId == 0L) {
```

Comment:
```suggestion
			if (seekBeforeId == Long.MAX_VALUE) {
```
In implementing `seekAfterId`, I optimized for 0 because 0 is the absolute minimum for punishment IDs. Here, the equivalent optimization would use Long.MAX_VALUE since Long.MAX_VALUE is the absolute maximum.

Reply:
Agreed, mistake which I oversaw sorry!